keyboards/nuphy/air75_v2/ansi/info.json

```diff
@@ -0,0 +1,275 @@
+{
+    "keyboard_name": "NuPhy Air75 V2",
+    "manufacturer": "NuPhy",
+    "usb":{
+        "vid": "0x19F5",
```

Comment:
This vendor ID appears to be in use/assigned already: 
https://www.the-sz.com/products/usbid/index.php?v=0x19F5

Reply:
I would like to clarify that we are authorized to use this VID by Nationz Technologies Inc. We are their customer and have been granted permission to use their VID for our products.
Thank you for your understanding and assistance with this matter.

Reply:
I don’t fully understand one of your suggestion. Currently, my code calls functions like setPinInput(pin) directly. Are you asking me to rewrap these functions, or should I replace them with the actual function calls, such as gpio_set_pin_input(pin)?

Reply:
They are not functions, they're C preprocessor macros, which have recently been renamed to better match the rest of the codebase: https://github.com/qmk/qmk_firmware/pull/23085

Reply:
@fauxpark I Got It, I have updated the code to replace the old macros with the new ones as per the recent changes in the codebase (#23085). Thank you for the clarification.